linera-service/src/cli_wrappers/wallet.rs

```diff
@@ -1376,39 +1376,37 @@ pub struct ApplicationWrapper<A> {
 }
 
 impl<A> ApplicationWrapper<A> {
-    pub async fn raw_query(&self, query: impl AsRef<str>) -> Result<Value> {
+    pub async fn json_query(&self, query: impl AsRef<str>) -> Result<Value> {
```

Comment:
`json` is not the right concept. I would suggest something like `run_graphql_query` or `send_graphql_query`

Reply:
Ok, I was unsure about the naming.